meta-mel/recipes-core/os-release/os-release.bbappend

```diff
@@ -0,0 +1,8 @@
+OS_RELEASE_FIELDS = "PRETTY_NAME NAME VERSION_ID VERSION VERSION_CODENAME ID HOME_URL SUPPORT_URL BUG_REPORT_URL BUILD_ID"
+
+ID = "flex-os"
+NAME = "${OS_NAME}"
+VERSION = "${DISTRO_VERSION}${@' (%s)' % DISTRO_CODENAME if 'DISTRO_CODENAME' in d else ''}"
+VERSION_ID = "${DISTRO_VERSION}"
+VERSION_CODENAME = "${@'(%s)' % DISTRO_CODENAME if 'DISTRO_CODENAME' in d else ''}"
+PRETTY_NAME = "${DISTRO_NAME} ${VERSION}"
```

Comment:
These aren't being defined with _mel or in mel.conf. If changes are made in a bbappend unconditionally, it makes the layer non-yocto-compliant. These need to either be moved to mel.conf or use the override. OS_RELEASE_FIELDS can also be easily appended rather than overridden entirely, no?

Reply:
Added the _mel pverrride. For OS_RELEASE_FIELDS the order is bit different so I completely override it.